Cognite.Simulator.Utils/ModelLibraryBase.cs

```diff
@@ -758,12 +770,8 @@ public interface IModelProvider<A, T>
         /// Returns the state object of the given version of the given model
         /// </summary>
         /// <param name="modelRevisionExternalId">Model revision external id</param>
+        /// <param name="skipCheck">If true, skips the check for filtering model revisions with processed status and filepath.</param>
         /// <returns>State object</returns>
-        Task<T> GetModelRevision(string modelRevisionExternalId);
-
-        /// <summary>
-        /// Delete all temporary model files stored in memory and on disk
-        /// </summary>
-        void WipeTemporaryModelFiles();
+        Task<T> GetModelRevision(string modelRevisionExternalId, bool skipCheck = false);
```

Comment:
`bool skipCheck = false` is not easy to undestand for the consumer of the method.
name it `includeUnextracted` or something that self-explains how the behaviour of the method changes, so the user doesn't need to read the docs/source.
change the doc string accordinly